com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipelineGlobalSettings.cs

```diff
@@ -897,6 +897,12 @@ internal bool AddDiffusionProfile(DiffusionProfileSettings profile)
 
 #endif
 
+        [SerializeField]
+        internal string DLSSProjectId = "231313132";
```

Comment:
Maybe use 000000 to highlight that it is not setup? (if it is allowed by the API)